spec/support/ansible_shared/automation_manager/refresher.rb

```diff
@@ -237,7 +237,7 @@ def assert_inventory_root_group
     expect(expected_inventory_root_group).to have_attributes(
       :name    => "hello_inventory",
       :ems_ref => "99",
-      :type    => "ManageIQ::Providers::AutomationManager::InventoryRootGroup",
+      :type    => manager_class::InventoryRootGroup.name,
```

Comment:
See https://github.com/ManageIQ/manageiq/pull/17574#issuecomment-403505086 for why I don't like just changing the spec test